pkg/relayer/proxy/proxy.go

```diff
@@ -187,3 +195,22 @@ func (rp *relayerProxy) validateConfig() error {
 
 	return nil
 }
+
+// PingAll tests the connectivity between all the managed relay servers and their respective backend URLs.
+func (rp *relayerProxy) PingAll(ctx context.Context) error {
+	var err error
+
+	for _, srv := range rp.servers {
+		if err := srv.Ping(ctx); err != nil {
+			err = errors.Join(err, err)
+		}
+	}
+
+	if err != nil {
+		rp.logger.Error().Err(err).
+			Msg("an unexpected error occured while pinging backend URL")
```

Comment:
I don't think an error message is necessary here, especially if we add more detail elsewhere as suggested.
```suggestion
		rp.logger.Error().Err(err).Send()
```